CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [2.5.2] - 2024-11-07
 
-Update luarocks and libraries version.
+- Update luarocks and libraries version.
```

Comment:
I disagree, this is just a release overview.

Reply:
What is "release overview message" and why is it in the change log?